string app_name
string text
string background_path
---
bool success
